README.md

```diff
@@ -34,9 +34,10 @@ At last, visit `localhost:3001`.
 To run locally in production mode run
 
     $ npm run dist
-    $ NODE_ENV=production npm start
+    $ NODE_ENV=production APP_ENV=[environment variable] npm start
```

Comment:
Maybe a combo. Such as `dev-start` is for `APP_ENV=development` so `NODE_ENV=production APP_ENV=development npm start` could be `prod-dev-start`. I just picked up the name came up in my mind here. However, it is just the icing on the cake, the PR itself is good enough.